[INFRA] Add more guidance in the PR template (#671)
process-analytics/bpmn-visualization-js: .github/pull_request_template.md

Explain the **details** for making this change: What existing problem does the Pull Request solve? Why is this feature beneficial?

> If you changed the rendering of the BPMN file, add a screenshot.

Reviewer: ```suggestion
If you have changed the rendering of the BPMN file, please add a screenshot.
```

> If you changed the rendering of the BPMN file, add a screenshot.

Reviewer 2: I would prefer not mentioning `file` as we are manipulating xml text source for the BPMN diagram.
```suggestion
If your contribution has changed the BPMN rendering, add a screenshot.
```

**On automatic closing of ISSUES**

Put `closes #XXXX` (remove backticks to let GitHub create the link) in your comment to auto-close the issue that your PR fixes (if such).

> :warning: Don't forget to add a comment on the issue so that we can assign it to you.

Reviewer: in theory it is mentioned before in CONTRIBUTING so maybe we can say:

If you haven't done it already, add a comment on the issue so that we can assign it to you.

Author: In theory, a contributor must read the contributing before start its PR.

But like @benjaminParisel does in this last PR, he does read it, and he finds it's easier to put this message here.
I don't think it'w a good idea to remind to read the contributing guide.

Reviewer: yes you are right, probably only individuals will attentively read contributing guide

warning should bring enough attention here (however if we have many contributors this will still not prevent the situation where 2 unattentive people are working on the same issue)

Reviewer 2: I would prefer @aibcmars's suggestion
We should mention at 1st in the template to read the contributing guide and ask the contributor he/she has followed it. GitHub also reminds the contributor by hightlighting the contributing and code of conduct file in the top right of its UI. Also remember that creating a PR can be done outside of the GitHub UI (web or mobile), for instance from IDE or GitHub CLI.

Second, when submitting the PR, it's too late to make us aware that someone is working on an issue: we know it once the PR is opened. We want to know it before the contributor starts working on the issue. If the reasons why we request that are uncleard, let's improve that.

Author: I changed the warning and its position.
